### DISCUSSION TITLE
GitHub Copilot - Error activate

### DISCUSSION THREAD

**Discussion by yahyrparedes** [community/community#107550] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

I start the purchase process and complete it satisfactorily, I have funds on the card
![Captura de pantalla 2024-02-11 a la(s) 10 50 53](https://github.com/community/community/assets/12869119/97f30e9b-0865-4aa3-ba45-94e9d9b8ac41)

Complet process payment success

![Captura de pantalla 2024-02-11 a la(s) 10 51 25](https://github.com/community/community/assets/12869119/434dc197-b5bc-4da6-b46e-3eab1df16386)

When I try to activate it in jetbrains I get this error

![Captura de pantalla 2024-02-11 a la(s) 10 53 09](https://github.com/community/community/assets/12869119/a059879d-cc4d-4913-a8ed-ad6025fbc898)

Pls Help me!

**Comment by davecheney** [comment, marked as answer]
@yahyrparedes i'm sorry but Jetbrains chat remains behind a waitlist due to the high level of interest in this product. You can join the waitlist here. https://github.com/github-copilot/chat_jetbrains_waitlist_signup/join
